packages/web3-validator/src/web3_validator.ts

```diff
@@ -101,7 +101,19 @@ export class Web3Validator {
 		}
 
 		const schemaKey = toHex(blake2b(utf8ToBytes(JSON.stringify(jsonSchema))));
+
 		if (!this._validator.getSchema(schemaKey)) {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+			if (jsonSchema?.items && jsonSchema.items.length > 0) {
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, no-plusplus
+				for (let i = 0; i < jsonSchema.items.length; i++) {
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+					if (jsonSchema?.items[i]?.$id === '') {
+						// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+						jsonSchema.items[i].$id = `${schemaKey}/items/${i}`;
```

Comment:
With something like the following, we can use `eslint-disable-next-line` only once:
```suggestion
			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
			const jsonSchemaItems = jsonSchema?.items;
			if (jsonSchemaItems && jsonSchemaItems.length > 0) {
				for (let i = 0; i < jsonSchemaItems.length; i++) {
					if (jsonSchemaItems[i]?.$id === '') {
						jsonSchemaItems[i].$id = `${schemaKey}/items/${i}`;
```

Reply:
thanks for the suggestion, Muhammad!

Reply:
You are welcome @nikoulai ,
However, I see that the code was not updated. Did you forget to push a local code?

Reply:
I made the change and reverted due to failure in the pipeline. Please see the action: https://github.com/web3/web3.js/actions/runs/4787005578/jobs/8511913375. If I misunderstood something I can open a small PR for the suggestion.

Reply:
The idea is to minimize the number of ignores by having an assigned variable once. Here is a working snippet:

```suggestion
			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
			const jsonSchemaItems = jsonSchema?.items as { $id: string }[];
			if (jsonSchemaItems && jsonSchemaItems.length > 0) {
				// eslint-disable-next-line no-plusplus
				for (let i = 0; i < jsonSchemaItems.length; i++) {
					if (jsonSchemaItems[i]?.$id === '') {
						jsonSchemaItems[i].$id = `${schemaKey}/items/${i}`;
```